Draft content moderation and a few updates for online-context (#207)
ournetworks/ournetworks.ca: code-of-conduct.md

## Scope

This Code of Conduct applies at the event, in the online spaces of the conference, and in public spaces when an individual is representing Our Networks.

## Content Moderation Guidelines

> Because of ensuring this code of conduct is enforcable in an online context across platforms, we have some guidelines about the types of spaces the conference includes.

Reviewer: ```suggestion
Because of ensuring this code of conduct is enforceable in an online context across platforms, we have some guidelines about the types of spaces the conference includes.
```

> Because of ensuring this code of conduct is enforceable in an online context across platforms, we have some guidelines about the types of spaces the conference includes.

Reviewer: ```suggestion
To ensure this Code of Conduct is enforceable in an online context across platforms, we have some guidelines about the types of spaces the conference includes.
```
Minor suggestion on flow

Reviewer 2: +1 @garrying 's re-wording

> * **Moderated** - one of the core team or a trained volunteer is present for the full duration of the event actively moderating. The platform provides at least a minimum set of tools for moderation.

Reviewer 2: ```suggestion
* **Moderated** - one of the organizers or a trained volunteer is present for the full duration of the event actively moderating. The platform provides at least a minimum set of tools for moderation.
```

> * **Partially moderated** - either the space is durational and cannot be continuously moderated (but a member of the core team or trained volunteer will check in periodically to check on participants and moderate if needed) and/or the space has some moderation tools but they may not be ideal or "complete"

Reviewer: ```suggestion
* **Partially moderated** - either the space is durational and cannot be continuously moderated (but a member of the core team or trained volunteer will check in periodically to check on participants and moderate if needed) and/or the space has some moderation tools but they may not be ideal or "complete".
```

> * **Partially moderated** - either the space is durational and cannot be continuously moderated (but a member of the core team or trained volunteer will check in periodically to check on participants and moderate if needed) and/or the space has some moderation tools but they may not be ideal or "complete".

Reviewer 2: ```suggestion
* **Partially moderated** - either the space is durational and cannot be continuously moderated (but an organizer or a trained volunteer will check in periodically to check on participants and moderate if needed) and/or the space has some moderation tools but they may not be ideal or "complete".
```

> * **Unmoderated** - the space cannot be moderated because of the nature of the platform/artwork/etc please use these spaces at your own risk/discretion, we will do our best to clearly mark them

Reviewer: ```suggestion
* **Unmoderated** - the space cannot be moderated because of the nature of the platform/artwork/etc please use these spaces at your own risk/discretion, we will do our best to clearly mark them.
```

> * **Unmoderated** - the space cannot be moderated because of the nature of the platform/artwork/etc please use these spaces at your own risk/discretion, we will do our best to clearly mark them.

Reviewer: ```suggestion
* **Unmoderated** - the space cannot be moderated because of the nature of the platform/artwork/etc. Please use these spaces at your own risk/discretion, we will do our best to clearly mark them.
```

Reviewer 2: Should we make clear something about reporting to us CoC issues in this context?

Author: I think these spaces are characterized by "even if someone reports it, we can't do anything"

> In order to be considered "moderateable" for the purposes of this conference, the minimum featureset we're requiring is that the platform allows Moderators who can mute participants (for audio), delete messages that have been posted (in text chats), or kick/ban partcipants if required. Users should be able to use a pseudonym, turn off their own microphone/camera, and leave at any time. Because of the limits of machine translation, the core team and volunteers are only able to moderate text written in English, unless arranged in advance.

Reviewer: ```suggestion
In order to be considered "moderateable" for the purposes of this conference, the minimum feature set we're requiring is that the platform allows Moderators who can mute participants (for audio), delete messages that have been posted (in text chats), or kick/ban participants if required. Users should be able to use a pseudonym, turn off their own microphone/camera, and leave at any time. Because of the limits of machine translation, the core team and volunteers are only able to moderate text written in English, unless arranged in advance.
```

> In order to be considered "moderateable" for the purposes of this conference, the minimum feature set we're requiring is that the platform allows Moderators who can mute participants (for audio), delete messages that have been posted (in text chats), or kick/ban participants if required. Users should be able to use a pseudonym, turn off their own microphone/camera, and leave at any time. Because of the limits of machine translation, the core team and volunteers are only able to moderate text written in English, unless arranged in advance.

Reviewer 2: ```suggestion
For a session to be considered "moderateable" for this conference, the minimum feature set we're requiring is that the platform or service allows:

- Moderators to: 
  - mute participants (for audio)
  - delete messages that have been posted (in text chats)
  - kick/ban participants if required
- Users to: 
  - use a pseudonym 
  - turn off their own microphone/camera 
  - leave at any time
  
  Because of the limits of machine translation, the organizers and volunteers are only able to moderate text written in English, unless arranged in advance.
```

Reviewer 2: Tried to break up the flow
